app/src/pages/outside/registrationPage/registrationPage.jsx

```diff
@@ -111,6 +112,12 @@ const TokenErrorSection = ({ tokenProvided }) => (
       <a className={cx('backlink')} href={referenceDictionary.rpLanding}>
         ReportPortal.io
       </a>
+      <br />
+      <FormattedMessage id={'RegistrationPage.or'} defaultMessage={'or '} />
+      <a className={cx('backlink')} href="#login">
```

Comment:
Please use `Link` from `redux-first-router-link` package to manage page traversing via redux.
See line 222 at [this](app/src/pages/outside/loginPage/pageBlocks/loginBlock/loginForm/loginForm.jsx) file f.e.